catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -364,6 +370,8 @@
         <property name="actionRegistry" ref="metacardActionRegistry"/>
         <property name="featureService" ref="featureService"/>
         <property name="endpointUtil" ref="endpointUtil"/>
+        <argument ref="queryResponseTransformer"/>
+        <argument ref="blueprintBundleContext"/>
```

Comment:
❓ Where is this `bluerprintBundleContext` coming from?

Reply:
Blueprint provides access to the Bundle Context, which is referenced by `blueprintBundleContext`